package.json

```diff
@@ -21,7 +21,8 @@
         "DocumentDB",
         "Graph",
         "Gremlin",
-        "MongoDB"
+        "MongoDB",
+        "PostgreSQL"
     ],
     "preview": true,
     "homepage": "https://github.com/microsoft/vscode-cosmosdb/blob/master/README.md",
```

Comment:
> We still need to update the repo name & GitHub links in code #1430

I don't know that I would say _need_ for something that's mostly engineering. We definitely want to be careful with this so we don't lose anything